developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md

```diff
@@ -0,0 +1,81 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+# Egeria Workshop Day 1
+### Introduction :
+ - [Agenda](https://github.com/odpi/egeria/wiki/Egeria-Meeting-1st,-2nd-October-2019-@-Huizen,-Netherlands)
+
+### Review changes to Asset zones and recent Asset management content
+- Asset Management definition
+- Avro File Support - different programming languages
+- ZoneMembership into classification: [1459](https://github.com/odpi/egeria/issues/1459)
+- Schema registry for EventBus stream support in future.
+### Data Engine Proxy: 
+- Config store using encrypted files [1646](https://github.com/odpi/egeria/issues/1646)
+- General cache/state persistence mechanism. [\#1622](https://github.com/odpi/egeria/issues/1622) 
+### Data Engine OMAS: 
+Whether we need to differentiate `Process Template` from `process`. Check conclusions on issue [\#1576](https://github.com/odpi/egeria/issues/1576)
+### Asset Lineage OMAS & Open Lineage Services
+- How can we build connector or in egeria for open lineage services.
+- Cassandra/Elasticsearch with JanusGraph for Lineage.
+- Security and encrypted credentials. Probably we can reuse same method from [1646](https://github.com/odpi/egeria/issues/1646)
+### UI Discussion
+- Authorization approach: token vs session. The conclusion is we will support both solutions for Egeria then companies can choose their own method.
+- We have agreed on the UI Exception strategy for all UI components
+### Dev-ops OMAS
+- physical vs software
+- cloud provider/infrastructure vs deployedSoftwareServer/software
+- [Digital service lifecycle](https://github.com/odpi/data-governance/tree/master/docs/data-privacy-pack#digital-service-lifecycle)
+- [Digital services](https://github.com/odpi/data-governance/tree/master/docs/digital-services)
+### Deduplication through stewardship server 
+- How can we detect duplicate metadata instances for the same physical assets.[\#1650](https://github.com/odpi/egeria/issues/1650)
+### Salesforce use case of CIM model
+- Join development open source with Linux.
+- JSON-LD scanned into open-metadata-archive.
+   - See area 5 models upcoming.
+   - see book on common information models by Mandy.
+### SAS
+- How SAS is creating metadata modelsOpen Source the SAS mappings
+- Possibility of unify existing SAS metadata model 
+- Prepackage of SAS container for evaluation purpose
+### Magic Box of Security
+- Column based security tag estimation
+- Location of the asset could change the confidentiality and not linked to asset ownership
```

Comment:
"Asset" has a relationship with "Location" - "AssetLocation". We agree that we don't use this relationship for the Magic box.

Reply:
Cool I will update this part.